WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'i18n-spec'
+
+Dir.glob(Rails.root.join('config', 'locales', '*.yml')).each do |locale_file|
+  RSpec.describe locale_file.to_s do
+    it { is_expected.to have_valid_pluralization_keys }
```

Comment:
We cannot use exactly that because of the `be_named_like_top_level_namespace`, which checks that the root key corresponds to the file name without the ".yml".
It would work for the main locales files, but not for the others.

Other than that we can use the other tests, it's worth nothing besides some more output :p